### PR TITLE
DOCS-956: Add Flutter code to drive rover in square tutorial

### PR DESCRIPTION
**Pull request by sguequierre**
Major thanks to @njooma for this code! 

- Direct users to follow flutter app tutorial to start
- Add new Base screen to flutter app with a button that drives the rover in a square, change robot screen to also load that if resource is base

### CODE REVIEW

**Review by npentrel — COMMENTED** [review]
This looks great, thank you for working on it. I think @JessamyT should review this as well - she's already got flutter set up and can test this.

File: docs/tutorials/get-started/try-viam-sdk.md [hunk 1]
```diff
@@ -195,6 +195,11 @@ If you successfully configured your robot and it is able to connect to the Viam
 If you open the developer console, you should see some output including the names of your rover's resources.
 These are the components and services that the robot is configured with in the Viam app.
 
+{{% /tab %}}
+{{% tab name="Flutter" %}}
+
+Flutter code must be launched from inside a running Flutter application, so to get started programming your rover with Flutter you should follow the instructions to [Build a Flutter App that Integrates with Viam](/tutorials/control/flutter-app/).
```
[the remark this answers is not in the document]
Comment (sguequierre) [reply docs/tutorials/get-started/try-viam-sdk.md:201]: Understood, thanks!

File: assets/tutorials/try-viam-sdk/button.png
[the remark this answers is not in the document]
Comment (sguequierre) [reply assets/tutorials/try-viam-sdk/button.png:1]: Done

File: assets/tutorials/try-viam-sdk/resource-menu.png
[the remark this answers is not in the document]
Comment (sguequierre) [reply assets/tutorials/try-viam-sdk/resource-menu.png:1]: Done

**Comment by JessamyT** [comment]
Okay I will try testing now. Can you add flutter to the language tags in the frontmatter of this tutorial?

**Review by JessamyT — APPROVED** [review]
Tested and it works great! Nice job. LGTM % tiny comments

File: docs/tutorials/get-started/try-viam-sdk.md [hunk 2]
```diff
@@ -473,6 +479,173 @@ async function main() {
...
+
+This imports the <file>base_screen.dart</file> file into the program and adds logic to check if a {{< glossary_tooltip term_id="resource" text="resource" >}} is "navigable", or, has a screen made for it.
+Base is the only resource that is navigable.
+
+To navigate to the base screen, save your code and launch your simulator.
+Navigate to the robot screen of a (live) machine with a base resource configured, and see the resources displayed like the following:
+
+{{<imgproc src="/tutorials/try-viam-sdk/resource-menu.png" resize="300x" declaredimensions=true alt="Machine resources listed in an example Flutter app">}}
+
+Then, click on the base to display <file>base_screen.dart</file>:
```
Comment (JessamyT) [inline docs/tutorials/get-started/try-viam-sdk.md:643]:
```suggestion
Then, click on the base to display the base screen.
You may need to scroll to the bottom of the list of resources.
```
[opt] I think it'd make more sense not to say it'll display the file but rather the screen the file creates

**Review by npentrel — APPROVED** [review]
This is great thank you both - ready when you've considered/implemented the last few comments

**Pull request merged**